src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -160,7 +153,7 @@ public void traverse() {
         }
         final String rowRestrictionWithFilterSetName = rowRestriction + " AND " + SchemaUtils.FILTER_SET_NAME + " = '" + filterSetName + "'";
 
-        boolean noVqslodFilteringRequested = (filterSetInfoTableRef == null);
```

Comment:
why wasn't this stopping the NPEs before?
        this.filterSetInfoTableRef = filterSetInfoTableName == null || "".equals(filterSetInfoTableName) ? null : new TableReference(filterSetInfoTableName, SchemaUtils.YNG_FIELDS);
        so we shouldn't be throwing it unless we are somehow setting the filterSetInfoTableName? Does this ternary need parens?